core/src/main/java/org/mule/runtime/core/message/Correlation.java

```diff
@@ -22,7 +18,7 @@
  * 
  * @since 4.0
  */
-public class MuleMessageCorrelation implements Serializable
+public class Correlation implements Serializable
```

Comment:
On second thought can we use CorrelationIdentifier?  I don't think Correlation really describes what it is..